### ISSUE TITLE
Automatically installed localised content: errors in metadata

### ISSUE THREAD

**Issue by smz** [joomla/joomla-cms#7280]
### Steps to reproduce the issue
- Perform Joomla! installation
- Select "Extra steps: Install languages"
- In "Install Language Packages" select one or more additional language
- Set "Activate the multilingual feature:" to "Yes"
- Select "Install localised content"
- When installation has completed login in the backend and examine the automatically installed articles

### Expected result
Articles should have valid metadata in "Publishing Options"

### Actual result
All metadata is inserted in the "Meta Description:" as a JSON encoded string: `{"robots":"","author":"","rights":"","xreference":""}`

### Additional comments
Unrelated, but I can't understand why when performing the "Extra steps: Install languages" the default value for "Activate the multilingual feature:" is "no". Wouldn't it be more logical that the default would be "Yes"?

**Comment by smz** [comment]
@infograf768 We have this small issue in multilingual installation...

**Comment by smz** [comment]
I think I found where the problem lies: /installation/model/languages.php at line 1127 we set `'metadesc'` to that value and we should probably set it to a null string.

I'm unsure if what we set at line 1125 for `'metadata'` is correct...

In that file we have another small issue: article title is set to the `PLG_ARTICLE_BUTTON_ARTICLE` localised constant. This might be OK for some languages, but it surely isn't for Italian where the translation is "Collega articolo' ("Link article"). We probably should introduce a new string constant...

I can write a PR for the above, but it must be decided if we want it for 3.4.2 (*then no new string constant should be introduced*) or for 3.5 (*then a new string constant *might* be introduced*).

**Comment by infograf768** [comment]
>Unrelated, but I can't understand why when performing the "Extra steps: Install languages" the default value for "Activate the multilingual feature:" is "no". Wouldn't it be more logical that the default would be "Yes"?

Nope. Most installs are not multilingual. They are monolingual and default language would be another language than en-GB, the one added to installation.

Agree concerning metadesc. Not
`			'metadesc'         => '{"robots":"","author":"","rights":"","xreference":""}',`
but
`			'metadesc'         => '',`

Introducing a new string in installation to replace "PLG_ARTICLE_BUTTON_ARTICLE" looks useless as anyway the article created is just a sample and might very soon be edited/replaced.

**Comment by infograf768** [comment]
But we could use: `COM_CONTENT_CONTENT_TYPE_ARTICLE` instead.
This would make:
```
		$newlanguage = new JLanguage($itemLanguage->language, false);
		$newlanguage->load('com_content.sys', JPATH_ADMINISTRATOR, $itemLanguage->language, true);
		$title = $newlanguage->_('COM_CONTENT_CONTENT_TYPE_ARTICLE`');
```

**Comment by smz** [comment]
This can be closed as we have a PR (#7281)
